crates/voicevox_core/src/core/infer/runtimes/onnxruntime.rs

```diff
@@ -47,7 +47,9 @@ impl InferenceRuntime for self::blocking::Onnxruntime {
     fn supported_devices(&self) -> crate::Result<SupportedDevices> {
         (|| {
             let cpu = CPUExecutionProvider::default().is_available()?;
-            let cuda = CUDAExecutionProvider::default().is_available()?;
+            let cuda = CUDAExecutionProvider::default()
+                .with_conv_algorithm_search(ort::CUDAExecutionProviderCuDNNConvAlgoSearch::Default)
+                .is_available()?;
```

Comment:
あ、今気付きましたが、ここだけは何も手を加えない方がいいかもです。

この`is_available`が何をしているのかというと、「ONNX Runtimeが静的に持っている名前のリストに名前が含まれているか探す」ということしかしていないはずなので。

```suggestion
            let cuda = CUDAExecutionProvider::default().is_available()?;
```

Reply:
今、確認して中身を見ましたが、確かにそうなってますね。修正します